CSM doc update (#440)
ObolNetwork/obol-docs: docs/advanced/lido_csm.md

Add the reward addresses of all cluster members. Choose whether the contract is immutable (reccommended option), whether to sponsor the maintainers of [splits.org](https://splits.org), and optionally whether to set a distribution bounty such that third parties could pay the gas costs of distributing the accrued rewards in exchange for a small fee.

:::tip

> If your cluster would like to contribute a portion of it's rewards to Obol’s 1% for Decentralization Fund, thereby earning [contributions](https://docs.obol.org/docs/faq/contributions), you can optionally add [retroactivefunding.obol.eth](https://etherscan.io/address/0xDe5aE4De36c966747Ea7DF13BD9589642e2B1D0d) to the splitter contract.

Reviewer: ```suggestion
If your cluster would like to contribute a portion of its rewards to Obol’s '[1% for Decentralisation](https://blog.obol.org/1-percent-for-decentralisation/)' Retroactive Fund, thereby earning [Obol Contributions](https://obol.org/contributions) as part of Lido's [DV Vault](https://research.lido.fi/t/integrate-csm-into-the-decentralized-validator-vault/8621), you must add [retroactivefunding.obol.eth](https://etherscan.io/address/0xDe5aE4De36c966747Ea7DF13BD9589642e2B1D0d) as a recipient of 0.1% of the splitter contract.
```

(FYI, use relative urls rather than full ones, for links on the existing site. Doesnt matter this time as i swapped the contributions link for the main site)

![CSM Extended Mode Settings](/img/CSM_walkthrough21.png)

> Sign the transaction. The cluster is ready for deposit from Lido CSM. At this point, you're job is finished.

Reviewer 2: ```suggestion
Sign the transaction. The cluster is ready for deposit from Lido CSM. At this point, your job is finished.
```

> :::info

Reviewer: ```suggestion
:::warning
```
A bit more clear than just an 'info'?

> When claiming your clusters rewards, be sure to claim in either native ETH or wstETH. Rebasing tokens like stETH may not receive the incremental yield you’re expecting. More information can be found in [0xSplits documentation](https://docs.splits.org/core/split#how-it-works).

Reviewer 2: ```suggestion
When claiming your cluster's rewards, be sure to claim in either native ETH or wstETH. Rebasing tokens like stETH may not receive the incremental yield you’re expecting. More information can be found in [0xSplits documentation](https://docs.splits.org/core/split#how-it-works).
```

> When claiming your cluster's rewards, be sure to claim in either native ETH or wstETH. Rebasing tokens like stETH may not receive the incremental yield you’re expecting. More information can be found in [0xSplits documentation](https://docs.splits.org/core/split#how-it-works).

Reviewer: ```suggestion
When claiming your cluster's rewards, be sure to claim in either native ETH or wstETH. Rebasing tokens like stETH may not receive the incremental yield you’re expecting. More information can be found in [Splits.org documentation](https://docs.splits.org/core/split#how-it-works).
```
